tasks/install_shell.sh

```diff
@@ -404,7 +404,7 @@ install_file() {
 
       rpm -Uvh --oldpackage --replacepkgs "$2"
       if test "$version" = 'latest'; then
-        yum install -y puppet-agent
+        yum install -y puppet-agent && yum upgrade -y puppet-agent
```

Comment:
Does this work for RedHat 6/7 as well?

Reply:
The task works as expected. AND 5.5.3 is available for both I guess I could add that to the acceptance test matrix.